src/func/funcCompile.ts

```diff
@@ -181,7 +187,7 @@ export async function funcCompile(args: {
         }
     } catch (e) {
         args.logger.error(e as Error);
-        throw Error("Unexpected compiler response");
+        return throwInternalCompilerError("Unexpected compiler response");
```

Comment:
@verytactical do you know any other way to fix linter error here? when I replaced `throw Error(..)` with `throwInternalCompilerError` it started showing errors in the next throw below it, and the only "good" way to fix it that I found was to add `return` here.